examples/documentation_examples/examples.go

```diff
@@ -2625,7 +2625,7 @@ func CausalConsistencyExamples(client *mongo.Client) error {
 }
 
 // RunCommandExamples contains examples of RunCommand operations.
-// Appears at https://www.mongodb.com/docs/manual/reference/command/collStats/.
+// Appears at https://www.mongodb.com/docs/manual/reference/command/buildInfo/.
 func RunCommandExamples(t *testing.T, db *mongo.Database) {
 	ctx := context.Background()
 
```

Comment:
Suggest removing code inserting documents into the `restaurants` collection.

I expect the output of `buildInfo` is not impacted by the documents in the `restaurants` collection.